### ISSUE TITLE
[Bug]: Why can't my computer connect to termux with ssh?

### ISSUE THREAD

**Issue by MrAXUAN** [termux/termux-app#3544]
### Problem description

I installed OpenSSH on termux, but my computer doesn't connect.

My phone with termux and computer are on the same wifi, and the computer cannot be connected through ssh, but I can connect again using juicessh on my phone with termux. What's going on?

### Steps to reproduce the behavior.

I installed openssh on termux.
`pkg upgrade`
`pkg install openssh`

After launching ssh, I looked at the IP address of my phone .
`sshd`
`ifconfig`

Then I used my computer to try to connect to termux with ssh.
`ssh u0_a350@192.168.188.129 -p 8022`
u0_a350 is my username and 192.168.188.129 It's my LAN IP.
I am sure that my computer and phone are in the same LAN.
After entering the command, the computer displays
`Connection timed out`

### What is the expected behavior?

My computer connects to termux smoothly.

### System information

* Termux application version: 0.118.0
* Android OS version: 11

**Comment by Inkrementator** [comment]
What's the phone model? I think I had the issue once with a Samsung galaxy S9, I assume it was a vendor specific firewall.

I worked around it by Remote Forwarding the port with SSH

From your phone:

```
ssh -R 8022:<PCPort> <YourPC>
```

and then on your PC

```
ssh -p <PCPort> localhost
```

to make it work

**Comment by Inkrementator** [comment]
Does this only happen when you enable the proxy-software? Which proxy software do you use. also,have you already tried setting up a remote forwarding?

**Comment by JackMorningStar** [comment]
Restart your phone and reinstall your termux, then definitely you will
reach your resolution.

**Comment by fatinghenji** [comment]
@Inkrementator Yes, this only happens after I have enabled the proxy software, I have tried 2, [FlClash](https://github.com/chen08209/FlClash) and [NekoBoxForAndroid](https://github.com/MatsuriDayo/NekoBoxForAndroid). On another question: I haven't tried it yet, I feel like the problem I have may be more complicated - I also run the proxy software on my computer.
As for the solution given by @JackMorningStar - it doesn't work. I installed/restarted it several times and the problem persisted.

**Comment by Inkrementator** [comment]
It looks like these apps work by posing as a VPN to android and thus being able to intercept all traffic. It is very likely that many operating systems for PCs handle inbound connections differently than android in this case, thus you can't really make useful deductions from it working there.

IF you can still make outbound connections to your local network from your phone while the proxy app is on, than I'm pretty sure that ssh remote forward will work.

A more elegant solution could be to disable termux in your proxy app, some allow you to do this.

https://github.com/MatsuriDayo/NekoBoxForAndroid/issues/752 this looks promising

**Comment by fatinghenji** [comment]
@Inkrementator I know what you're talking about, but this way I can't use proxies in termux. Especially in China, I can't access many sources normally. It would take a lot of time to manually configure them one by one. Is it possible to use a method like '-x 127.0.0.1:7890' for the entire termux environment to do "pseudo-global" proxies?
